Geography/Urban Rural Classification/1_Create Urban Rural Classification Lookup Files.R

```diff
@@ -0,0 +1,224 @@
+#
+# install.packages("magrittr")
+# install.packages("tidyr")
+# install.packages("dplyr")
+# install.packages("readxl")
+# install.packages("readr")
+# install.packages("tidylog")
+# install.packages("glue")
+#
+# Description - Creating Urban-Rural Classificiation Lookup Files
+# Approximate run time - 22 seconds
+
+base_filepath <- file.path("//Freddy", "DEPT", "PHIBCS", "PHI", 
+                           "Referencing & Standards", "GPD", "1_Geography", 
+                           "Urban Rural Classification")
+data_filepath <- file.path(base_filepath, "Source Data", "2016")
+lookups_filepath <- file.path(base_filepath, "Lookup Files", "R Files")
+
+# Read in packages from library
+library(magrittr)
+library(tidyr)
+library(dplyr)
+library(readxl)
+library(readr)
+library(tidylog)
+library(glue)
+
+### 2 - Create Lookup for Urban Rural Columns ----
+
+UR_Lookup <- read_csv(glue("{data_filepath}/DZ2011_SGUR2016_Lookup.csv")) %>% 
+  select(-DZ_CODE) %>% 
+  distinct() %>% 
+  rename(UR2_2016 = UR2FOLD, 
+         UR3_2016 = UR3FOLD, 
+         UR6_2016 = UR6FOLD, 
+         UR8_2016 = UR8FOLD) %>% 
+  mutate(UR2_2016_name = case_when(UR2_2016 == 1 ~ "1 Urban Areas", 
+                                   UR2_2016 == 2 ~ "2 Rural Areas"),
+         UR3_2016_name = case_when(UR3_2016 == 1 ~ "1 Rest of Scotland", 
+                                   UR3_2016 == 2 ~ "2 Accessible Rural",
+                                   UR3_2016 == 3 ~ "3 Remote Rural"),
+         UR6_2016_name = case_when(UR6_2016 == 1 ~ "1 Large Urban Areas", 
+                                   UR6_2016 == 2 ~ "2 Other Urban Areas",
+                                   UR6_2016 == 3 ~ "3 Accessible Small Towns",
+                                   UR6_2016 == 4 ~ "4 Remote Small Towns", 
+                                   UR6_2016 == 5 ~ "5 Accessible Rural",
+                                   UR6_2016 == 6 ~ "6 Remote Rural"),
+         UR8_2016_name = case_when(UR8_2016 == 1 ~ "1 Large Urban Areas", 
+                                   UR8_2016 == 2 ~ "2 Other Urban Areas",
+                                   UR8_2016 == 3 ~ "3 Accessible Small Towns",
+                                   UR8_2016 == 4 ~ "4 Remote Small Towns", 
+                                   UR8_2016 == 5 ~ "5 Very Remote Small Towns",
+                                   UR8_2016 == 6 ~ "6 Accessible Rural",
+                                   UR8_2016 == 7 ~ "7 Remote Rural", 
+                                   UR8_2016 == 8 ~ "8 Very Remote Rural")) %>% 
+  select(UR2_2016, UR2_2016_name, UR3_2016, UR3_2016_name, UR6_2016, 
+         UR6_2016_name, UR8_2016, UR8_2016_name)
+
+
+
+### 3 - Create DataZone2011 to Urban Rural Lookup ----
+
+# Get DZ2011_SGUR2016_Lookup file
+# Rename variables and sort by DataZone2011
+
+DZ2011_SGUR2016_Lookup <- read_csv(glue("{data_filepath}/", 
+                                        "DZ2011_SGUR2016_Lookup.csv")) %>% 
+  rename(DataZone2011 = DZ_CODE, 
```

Comment:
Shall we use all lower case for the column names in the future to keep things consistent?